pkg/migrations/migration_version.go

```diff
@@ -17,6 +18,7 @@ const (
 	// MigrationVersionFile records the latest central version in databases.
 	MigrationVersionFile     = "migration_version.yaml"
 	migrationVersionFileMode = 0644
+	lastRocksDBVersion       = "3.74.0"
```

Comment:
How do you get this implication? Or I shall name it lastRocksDBMajorVersion.

Reply:
So I think ideally for a rollback from 4.0P to 3.74R the idea would be for them to rollback to the version that is in the RocksDB current.  if the customer is coming from `3.74.2` and we store in `3.74.0` then the versions in current won't match and the rollback will fail.  I think the idea was that a rollback from 4 back to 3.74 would be simplified and not require the forceRollbackVersion to be set which would require us storing the version as to what the customer was coming from.

Reply:
3.72 -> 4.0 -- Rocks will need to be migrated before rocks to postgres migrations can be performed.  How does this change handle that?  Would current still be 3.72?  We would have to not persist in addition to not setting the version to accomplish that.  The other question is in that case would we want Rocks to be migrated from 3.72 -> 3.74.  that situation would be complicated.  

Might be best to also not persist.  Rocks migrations if necessary would be performed against temp and then temp would be used to migrate to Postgres.  But keep current at the old version and then a rollback to the old version would be seamless.  In theory

Reply:
First for 3.72 -> 4.0, first we do not recommend it. 
If it happens, if the pvc has enough storage, it would end up having 1) .previous with 3.72 and 2) current with 3.74.0 (faked) with upgrade DB ver 112. 
So the customer would still have the ability to rollback as earlier upgrades.
if the pvc does not have enough storage, it would end up having 1) current with 3.74.0 (faked) with upgrade DB ver 112. 
The customer do not have the ability to rollback to 3.72 which is the situration as other upgrades without enough PVC but since we have a current with 3.74.0, the customer can rollback to 3.74.0 or just directly upgrade it to any other 3.74 release if needed.

Reply:
i'm probably going to have to pull this down and run some tests.  I think stuffing 3.74.0 in there causes issues as we either have to keep that up to date or have special logic to handle a rollback to 3.74.0 when we really mean 3.74.2.

Lets look at another scenario.  Say the upgrade from 3.73.2 -> 4.0.  If we just don't persist current or write the migration yaml file then a rollback will just go right back to the current which would be 3.73.2.  If we put 3.74 in there then they are rolling back to a release they were never on.  I think trying to create fake versions makes things more confusing and harder to maintain.

Reply:
Sure. 
We do not need to update this version. It is purposely put between 3.73 and 3.74 to the the lowest version that is recommended to upgrade from to 4.0. 
3.73.2R is in the same condition with 3.72.
It is not an issue if customer has been on 3.74.0 or not because the schema is frozen and has no change in 3.74. The customer or support is not required to fill in this version into the config map for rollback. The expected behavior does not change except in case of no way to recover, we can still upgrade to 3.74.0 and beyond.